docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -46,11 +46,11 @@ and the scheduler:
 --feature-gates=PodPriority=true
 ```
 
-Also set this flag for API server:
+Also enable scheduling.k8s.io/v1alpha1 API and Priority admission controller in API server:
 
 
 ```
---runtime-config=scheduling.k8s.io/v1alpha1=true
+--runtime-config=scheduling.k8s.io/v1alpha1=true --admission-control=Priority
```

Comment:
I'd suggest we tell the users how to set up the admission control with concrete/accurate examples.

One way of doing this is to show an example which is kind of "best practice".
Regarding '--admission-control', the "best practice" could be this (borrowed from https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/controlplane/manifests.go#L45):

```
--admission-control="Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota"
```

However, different versions have different ”sequence" (not "set") of recommended plugins to enable. It may worth a standalone section for admission controller.

Another option is to express this in a vague way, for example, "make sure `Priority` is included in the comma separated value for the `--admission-control` flag, i.e. `--admission-control=Foo,Bar,Priority,Zoo...". A native speaker may help polish this statement.

Reply:
Done.